ui/src/tasks/components/TaskRunsPage.tsx

```diff
@@ -49,7 +51,9 @@ class TaskRunsPage extends PureComponent<Props> {
         <Page titleTag="Runs">
           <Page.Header fullWidth={false}>
             <Page.Header.Left>
-              <PageTitleWithOrg title="Runs" />
+              <PageTitleWithOrg
+                title={`${_.get(currentTask, 'name', '')} - Runs`}
```

Comment:
Should the `- ` be there if the name lookup fails? Might look weird having ` - Runs` as a title.

Reply:
I updated it with a getter for title